script.skin.info.service/Utils.py

```diff
@@ -0,0 +1,286 @@
+import xbmc
+import xbmcaddon
+import xbmcgui
+import os
+import sys
+import json as simplejson
```

Comment:
can you remove the alias to simplejson and simply replace everything with `json`?